examples/advanced/federated-statistics/image_stats/jobs/image_stats/custom/image_statistics.py

```diff
@@ -94,7 +94,7 @@ def histogram(
     ) -> Histogram:
         histogram_bins: List[Bin] = []
         histogram = np.zeros((num_of_bins,), dtype=np.int64)
-
+        bin_edges = []
```

Comment:
If number of bins is not 0,

This bin-edges[j] might get index out of bound error